File: sixth-exercises/c/4.c
#include <stdio.h>
#include <string.h>

int main() {
    char full_name[100];
    char *first_name, *last_name;

    printf("Digite um nome completo: ");
    fgets(full_name, sizeof(full_name), stdin);
    
    last_name = strrchr(full_name, ' ');
    if (last_name != NULL) {
        *last_name = '\0';  // Terminate the last name.
        last_name++;        // Move to the first character of the first name.
        printf("%s/%s\n", last_name, full_name);
    } else {
        printf("Nome completo inválido.\n");
    }

    return 0;
}

File: sixth-exercises/python/4.py
def format_name(full_name):
    name_parts = full_name.split()
    if len(name_parts) >= 2:
        last_name = name_parts[-1]
        first_name = ' '.join(name_parts[:-1])
        return f"{last_name.upper()}/{first_name}"
    else:
        return "Nome completo inválido."

full_name = input("Digite um nome completo: ")
formatted_name = format_name(full_name)
print("Sobrenome/Nome formatado: ", formatted_name)
